rainier-core/src/main/scala/com/stripe/rainier/core/Discrete.scala

```diff
@@ -67,7 +67,7 @@ final case class Geometric(p: Real) extends Discrete {
   * @param n Total number of failures
   * @param p Probability of success
   */
-final case class NegativeBinomial(n: Real, p: Real) extends Discrete {
+final case class NegativeBinomial(p: Real, n: Real) extends Discrete {
```

Comment:
I switched the order so that it is the same as `Binomial` (even though I think both should be the other way :shrug:).